sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantPreAuthSettings.md

```diff
@@ -0,0 +1,296 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spotenantpreauthsettings
+applicable: SharePoint Online
+title: Set-SPOTenantPreAuthSettings
+schema: 2.0.0
+author: lw-msft
+ms.author: laurenwong
+ms.reviewer:
+manager: bhaveshd
+---
+
+# Set-SPOTenantPreAuthSettings
+
+## SYNOPSIS
+
+Sets the pre auth settings for the tenant. 
+
+**What is pre auth?**
```

Comment:
this header is not allowed. See the [platyPS schema](https://github.com/PowerShell/platyPS/blob/master/docs/developer/platyPS/platyPS.schema.md) for what's expected.

Options:
1. Remove the header, and directly explain what pre auth is.
2. Have a separate documentation for pre auth, and link it here.